release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -74,6 +74,8 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 [id="ocp-4-11-installation-and-upgrade"]
 === Installation and upgrade
 
+This release updates the minimum system requirements for installing OpenShift on a single node. Users should configure a minimum of 16 GB of RAM. Specific workload requirements may require additional RAM. The complete list of supported platforms has been updated to include bare metal, vSphere, Red Hat OpenStack, and Red Hat Virtualization platforms. In all cases, the user must specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file.
```

Comment:
is it clear that that 2nd sentence refers to SNO?
Also, I would add something like:

The user must specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file in a case that openshift-installer binary is being used to install single node OpenShift.

Reply:
Updated as suggested.